README.md

```diff
@@ -33,75 +33,112 @@ Confused about the terminology of APM? Take a look at the [APM Glossary][visuali
-[docker-compose]: https://www.docker.com/products/docker-compose
 
-#### Set up Python
+#### Python
 
-1. Clone the repository locally: `git clone https://github.com/DataDog/dd-trace-py`
-2. The tests for this project run on various versions of Python. We recommend
-   using a Python version management tool, such as
-   [pyenv](https://github.com/pyenv/pyenv), to utilize multiple versions of
-   Python. Install Pyenv: https://github.com/pyenv/pyenv#installation
-3. Install the relevant versions of Python in Pyenv: `pyenv install 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
-4. Make those versions available globally: `pyenv global 3.9.1, 2.7.18, 3.5.10, 3.6.12, 3.7.9, 3.8.7, 3.10.0`
+The library supports multiple Python runtime versions and platforms. See [supported runtimes](https://ddtrace.readthedocs.io/en/stable/versioning.html#supported-runtimes) for the full list. You have two options for preparing a local development environment. You can use pyenv to install Python runtimes on your local machine or you can use the `datadog/dd-trace-py` Docker image which includes all runtimes.
 
-### Testing
+##### Option #1: pyenv
+
+1. [Install pyenv](https://github.com/pyenv/pyenv#getting-pyenv)
+2. Install Python runtimes with pyenv
+3. [Optional] [Install pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv)
+
+To install all the tested Python runtimes:
 
-#### Running Tests in docker
+    cd docker; pyenv local | xargs -L 1 pyenv install
+    
+We recommend using a virtual environment for local development. 
 
-Once your docker-compose environment is running, you can use the shell script to
-execute tests within a Docker image. You can start the container with a bash shell:
+If you followed the optional step to install the `pyenv-virtualenv` plugin, you can use it to manage your Python virtual environments:
 
-    $ scripts/ddtest
+    pyenv virtualenv 3.9.11 ddtrace-py3.9
+    pyenv activate ddtrace-py3.9
 
-You can now run tests as you would do in your local environment. We use
-[tox][tox] as well as [riot][riot], a new tool that we developed for addressing
-our specific needs with an ever growing matrix of tests. You can list the tests
-managed by each:
+Before running tests, you will need to install the following test requirements:
+
+    pip install riot tox 
+
+We use [tox][tox] as well as [riot][riot], a new tool that we developed for
+addressing our specific needs with an ever growing matrix of tests. You can list
+the tests managed by each:
 
     $ tox -l
     $ riot list
 
-You can run multiple tests by using regular expressions:
+We include a helper script to run all tox environments matching a regular expression:
 
     $ scripts/run-tox-scenario '^futures_contrib-'
-    $ riot run psycopg
-
+    
 [tox]: https://github.com/tox-dev/tox/
 [riot]: https://github.com/DataDog/riot/
 
-#### Running Tests locally
+##### Option #2: datadog/dd-trace-py
+
+Use provide a script to start the
```

Comment:
Not sure what this sentence means (I'm not a native English speaker).
Maybe reformulate as: "Use the following script..."?

Reply:
```suggestion
Use the provided script to start the
```

I bet this is what was intended